doc/cephfs/multimds.rst

```diff
@@ -40,7 +40,7 @@ result of commands.
 
     # fsmap e5: 1/1/1 up {0=a=up:active}, 2 up:standby
 
-    ceph fs set max_mds 2
+    ceph fs set <cephf-name> max_mds 2
```

Comment:
might want to use <fs_name> to be consistent with https://github.com/ceph/ceph/blob/master/doc/man/8/ceph.rst#fs.

also, in the context of cephfs, it's a little bit redundant to put `cephfs-name` here.